### PR TITLE
Add moz-fluent-linter to pre-commit 

### PR DESCRIPTION
**Pull request by robhudson**
## One-line summary

This linter runs during our pull requests, so this can help catch things earlier.

The only thing I'm sure about is what to do with the other l10n linting we do? There's 4 different linting commands being run in `./bin/run-tests.sh`. Can we remove any with this addition?

## Testing

Run `pre-commit run --all-files`. Make a change to an `.ftl` file that causes an error, like using a capital letter or underscore in a string identifier and run.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -73,6 +73,12 @@ repos:
     hooks:
       - id: black
         language_version: python3
+  - repo: https://github.com/mozilla-l10n/moz-fluent-linter
+    rev: v0.4.4
+    hooks:
+      - id: fluent_linter
+        files: \.ftl$
+        args: [--config, .github/l10n/linter_config.yml, l10n/en/]
```
Comment (pmac) [inline .pre-commit-config.yaml:81]: we also have FTL files in `l10n/en-US` and `l10n-pocket/en`. Could we add those?

**Comment by robhudson** [comment]
Waiting on a small update to the hook before pushing this further:
https://github.com/mozilla-l10n/moz-fluent-linter/pull/12

**Comment by robhudson** [comment]
Ready for merge

**Review by stevejalim — APPROVED** [review]
r+ Nice work and an elegant solution to our fluent linting gaps

**Pull request merged**